README.md

```diff
@@ -537,26 +536,7 @@ We pass all tests in https://github.com/bradfitz/go-sql-test using go-mysql driv
 
 ## Logging
 
-Logging by default is send to stdout.
-
-To disable logging completely:
-```go
-import "github.com/siddontang/go-log/log"
-...
-        nullHandler, _ := log.NewNullHandler()
-        cfg.Logger = log.NewDefault(nullHandler)
-```
-
-To write logging to any [`io.Writer`](https://pkg.go.dev/io#Writer):
-```go
-import "github.com/siddontang/go-log/log"
-...
-        w := ...
-        streamHandler, _ := log.NewStreamHandler(w)
-        cfg.Logger = log.NewDefault(streamHandler)
-```
-
-Or you can implement your own [`log.Handler`](https://pkg.go.dev/github.com/siddontang/go-log/log#Handler).
```

Comment:
The slog is more complex to be implemented at the user side. Please leave the guidance in readme https://github.com/golang/example/blob/master/slog-handler-guide/README.md

Reply:
I generally expect people will use an existing adapter like https://github.com/samber/slog-zerolog or https://github.com/samber/slog-zap

Reply:
something like https://github.com/serprex/slog-siddontang

Reply:
Thanks! I'll take a look soon.